gatsby-node.js

```diff
@@ -79,7 +81,115 @@ exports.createPages = async ({ graphql, actions }) => {
       }
     }
   `)
+  
+  createMeetupsByYearPage(result, { createPage })
+
+  await createMeetupPosters(result)
+}
+
+const dataMappersByType = {
+  'meetup': prepareDataForMeetupTemplate,
+  'meetup-hero': prepareDataForMeetupHeroTemplate,
+}
+
+async function createMeetupPosters (result) {
+  console.log('⏳ Start creating posters')
+  const meetups = result.data.allMarkdownRemark.edges.map(({ node }) => {
+    return {
+      ...node.frontmatter,
+      type: node.frontmatter.type ? node.frontmatter.type : 'meetup',
```

Comment:
On pourrait pas utiliser un `||` ou un `??` ?

```suggestion
      type: node.frontmatter.type || 'meetup',
```